OneCommunityGlobal/HighestGoodNetworkApp
# Sucheta - Update Projects Page sorting (by Project Name) and Filters Project List by Category Name and Active/ Inactive

# Description
(PRIORITY HIGH) Jae: Update Projects Page sorting (WIP Sucheta) 
Other Links>Projects
I’d like the default filter/sorting of this page to show the Projects that have most recently had additions to their WBSs at the top. 
2ND PR: I’d also like sort buttons added by Project Name, Category, and Active so I can 1-click and sort by these instead if needed
3RD PR: Add a sort button to the Members and Inventory heading so I can 1-click and sort by whichever of these have had edits most recently. 


## Related PRS (if any):
Backend [PR#693](https://github.com/OneCommunityGlobal/HGNRest/pull/693)
…

## Main changes explained:
- Adds a `DropdownButton` component using ReactBootstrap to `ProjectTableHeader.jsx` - `Category ` and `Active `.
- Project Name can be sorted alphabetically.

…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5. go to dashboard→ Projects
6. The table header should have sorting buttons by the Projects Name. The `downArrow ` sorts the project list (A-Z) and `upArrow ` sorts (Z-A).
7. The `Category ` has a dropDown to choose between categories. To reset the selected category, simple select `Unspecified` from the dropdown.
8. The `Active` has a dropdown to choose between active and inactive projects. To reset simply choose "Choose status".

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/101291283/73e9de4b-4ad5-425c-8e06-3491552df560



## Note:
Still working on the Sort functionality that will bring the most recently modified projects to the top of the list.
